<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(95, 210, 181, 10)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushStart">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>verdana</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;verdana&quot;;
/*background-color: rgb(249, 222, 149);*/
background-color: rgb(255, 255, 255);
border-radius: 8px;
border: 2px solid #9a9;
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5);
selection-color: rgb(170, 255, 127);</string>
    </property>
    <property name="text">
     <string>СТАРТ</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushStatistic">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Constantia&quot;;
background-color: rgb(249, 222, 149);
</string>
    </property>
    <property name="text">
     <string>СТАТИСТИКА</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushClean">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Constantia&quot;;
background-color: rgb(249, 222, 149);
</string>
    </property>
    <property name="text">
     <string>ОЧИСТИТЬ ПРОГРЕСС</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Constantia&quot;;</string>
    </property>
    <property name="text">
     <string>version 1.0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAbout">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 222, 149);
font: 13pt &quot;Constantia&quot;;
</string>
    </property>
    <property name="text">
     <string>О ПРОГРАММЕ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="D:/Graphics/Avto/KIA/1-full.jpg"/>
 </resources>
 <connections/>
</ui>
